app/screens/TransactionAuthorization.tsx

```diff
@@ -184,6 +185,12 @@ export function TransactionAuthorization (): JSX.Element | null {
         .then(async signedTx => {
           // case 1: success
           await resetPasscodeCounter()
+          if (transaction?.title != null && transaction?.description != null) {
```

Comment:
Better move it to `OceanInterface`. So we can trigger it during:
1. Before broadcast
2. On completion of broadcast
3. If there are any errors